polkadot/node/core/pvf/src/testing.rs

```diff
@@ -100,24 +100,6 @@ pub fn build_workers_and_get_paths() -> (PathBuf, PathBuf) {
 		let mut execute_worker_path = workers_path.clone();
 		execute_worker_path.push(EXECUTE_BINARY_NAME);
 
-		// explain why a build happens
```

Comment:
I just removed this because the explanation is only ever shown when `-- --nocapture` is passed in, and even then it does not explain a rebuild most of the time. This code would have needed to be updated, and I didn't feel like updating un-useful code.